python/filtering/filter_menu.py

```diff
@@ -1141,7 +1141,7 @@ def _get_search_filter_field_id(filter_id):
         :return: The field id that the search filter item widget refers to.
         :rtype: str
         """
-        return re.sub(rf".{str(SearchFilterItemWidget)}$", "", filter_id)
+        return re.sub(r"{}$".format(str(SearchFilterItemWidget)), "", filter_id)
```

Comment:
Caught another Syntax error from this line` rf".{str(SearchFilterItemWidget)}$"`.  Everything's working fine now.